test/maverick/node/node.go

```diff
@@ -685,6 +701,7 @@ func NewNode(config *cfg.Config,
 	genesisDocProvider GenesisDocProvider,
 	dbProvider DBProvider,
 	metricsProvider MetricsProvider,
+	dashCoreRpcClient *dashcore.RpcClient,
```

Comment:
Most nodes will not have a dash core rpc client, so it shouldn't be passed as a parameter here, but instead created if `if config.PrivValidatorCoreRPCHost != "" {` and then set to the node.